src/pages/index.jsx

```diff
@@ -427,7 +432,7 @@ const QuickstartsPage = ({ data, location }) => {
             </Overlay>
           </div>
 
-          {isSearchInputEmpty && (
+          {isSelectCategory && isSearchInputEmpty && (
```

Comment:
This approach totally works, it just adds some complexity with a new hook. Another option would be to key off of the `category` hook, which is already being/unset by the existing code. Then you can completely remove the need for a new `isSelectCategory` hook.
```suggestion
          {category === '' && isSearchInputEmpty && (
```